setup.py

```diff
@@ -36,22 +36,27 @@
     "raven==5.0.0",  # FIXME: To be compatible with our sentry version
     "orcid",
     "retrying",
-    "invenio-query-parser>=0.3.0",
     "invenio_classifier==0.1.0",
-    "invenio-access==0.1.0",
-    "invenio-client==0.1.0",
-    "invenio-formatter==0.2.1",
-    "invenio-groups==0.1.3",
+    "invenio-access==0.2.0",
```

Comment:
Above, in base.requirements.txt, we are using the `master` branch, so if that is needed this line should be removed.